src/renderer/components/dex/OrderHistory.vue

```diff
@@ -14,9 +14,9 @@
                 <th>{{$t('pairLc')}}</th>
                 <th>{{$t('filled')}}</th>
                 <th>{{$t('unitsTotal')}}</th>
-                <th>{{$t('priceLc')}}</th>
+                <th class="price">{{$t('priceLc')}}</th>
```

Comment:
Generally, we want to use a space inside on each side of the curly braces. Ex.  {{ $t('priceLc') }}

Reply:
I will change.